bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -41,28 +49,52 @@
  * @author Markus Rathgeb - Initial contribution and API
  * @author Mark Herwege - Added methods to find broadcast address(es)
  * @author Stefan Triller - Converted to OSGi service with primary ipv4 conf
+ * @author Gary Tse - Network address change listener
  */
 @Component(configurationPid = "org.eclipse.smarthome.network", property = { "service.pid=org.eclipse.smarthome.network",
         "service.config.description.uri=system:network", "service.config.label=Network Settings",
-        "service.config.category=system" })
+        "service.config.category=system", "service.config.network.interface_poll.frequency_seconds=120" })
 @NonNullByDefault
 public class NetUtil implements NetworkAddressService {
 
     private static final String PRIMARY_ADDRESS = "primaryAddress";
     private static final String BROADCAST_ADDRESS = "broadcastAddress";
+    private static final String INTERFACE_POLL_FREQUENCY_SECONDS_KEY = "service.config.network.interface_poll.frequency_seconds";
```

Comment:
Don't you think that this is a pretty weird key if you compare it with `primaryAddress` and `broadcastAddress`?

Reply:
Renamed; other suggestions welcomed.

Reply:
How about `pollingInterval`? Btw, the constant should also be renamed from FREQUENCY to INTERVAL, because that's what it actually is.

Reply:
Thanks, addressed